src/api/app/controllers/person_controller.rb

```diff
@@ -281,4 +283,10 @@ def delete_token
     token.destroy
     render_ok
   end
+
+  private
+
+  def load_config
+    @configuration = ::Configuration.first
```

Comment:
Why do we store this in an instance variable? We could and probably should calle it directly were we need it. Especially since we use it only once.

I hope this wasn't requested by someone else;-)

Also you can just call ``` Configuration.passwords_changable? ```. No need to fetch the first Configuration instance first.